src/components/Card/__stories__/CardShowcase.scss

```diff
@@ -0,0 +1,57 @@
+.card-showcase-stories:not(:first-child) {
+    margin-left: 24px;
+}
+
+.card-showcase-stories {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 120px;
+    height: 120px;
+
+    &__desc {
+        text-align: center;
+        font-size: var(--yc-text-body-font-size);
+        line-height: var(--yc-text-body-line-height);
+        color: var(--yc-color-text-complementary);
+        margin-top: 3px;
+    }
+}
+
+.panel-showcase-stories {
```

Comment:
That class can be named as `.card-showcase` as it used as a container for examples

Reply:
I've refactored stories 6c24cf8